dockerfile-latest

```diff
@@ -0,0 +1,41 @@
+FROM airsharedcontainers.azurecr.io/vstsdropdownloadcore:master AS app
+FROM airsharedcontainers.azurecr.io/msftkube AS msftkube
+FROM microsoft/vsts-agent:ubuntu-16.04-docker-17.12.0-ce-standard
+
+# Get vstsdropdownloadcore's built drop downloader application
+copy --from=app /app/* /app/
+copy --from=msftkube /msftkube/src/dist/*.wh /msftkube/package.wh
+
+RUN apt-get -qq update
+
+# Install Python2.7 and Python2.7 dependencies
+RUN apt-get -qq install -y --no-install-recommends \
+                python2.7 \
+                python-pip \
+                python-dev
+
+RUN wget -q https://bootstrap.pypa.io/get-pip.py
+RUN python2.7 get-pip.py
+
+RUN python2.7 -m pip -q install pip==9.0.3
+RUN python2.7 -m pip -q install --upgrade setuptools
+RUN python2.7 -m pip -q install --upgrade azure azure-cli
+
+# Install Python3 and Python 3 dependencies
+RUN rm -rf ~/.cache/pip
+RUN apt-get -qq install -y --no-install-recommends \
+                python3 \
+                python3-pip
+
+RUN python3 -m pip -q install --upgrade pip
+RUN python3 -m pip -q install --upgrade setuptools
+RUN python3 -m pip -q install pyyaml~=4.2b4
```

Comment:
why are these two (pyyaml and azurekeyvault).  called out seperately? just a comment would be helpful If you don't know say what fails when you remove them.

Reply:
This was to fix a dependency issue (because it pull in a conflicting version as part of an install of another package otherwise)

Reply:
removed